aptos-move/framework/aptos-stdlib/sources/simple_map.move

```diff
@@ -77,6 +91,18 @@ module aptos_std::simple_map {
         vector::push_back(&mut map.data, Element { key, value });
     }
 
+    /// Add multiple key/value pairs to the map. The keys must not already exist.
+    public fun add_all<Key: store, Value: store>(
```

Comment:
batch_add?

Reply:
I prefer add_all as it's more consistent with add